iidm/iidm-serde/src/test/java/com/powsybl/iidm/serde/AbstractIidmSerDeTest.java

```diff
@@ -116,11 +116,11 @@ protected void allFormatsRoundTripFromVersionedXmlFromMinToCurrentVersionTest(St
     /**
      * Execute a round trip test reading the test resource IIDM-XML file with a given file name comparing
      * the output IIDM-XML file to a reference file for all IIDM versions equals or more recent than
-     * a given minimum IIDM version <b>and</b> strictly older than the current IIDM version.
+     * a given minimum IIDM version <b>and</b> strictly older than V_1_13.
      */
-    protected void allFormatsRoundTripFromVersionedXmlFromMinToCurrentVersionTest(String file, String reference, IidmVersion minVersion) throws IOException {
+    protected void allFormatsRoundTripFromVersionedXmlFromMinToVersion1Dot13Test(String file, String reference, IidmVersion minVersion) throws IOException {
```

Comment:
I think it would be cleaner to have the max version in parameter. This way, we won't have to create a new method for each max version.